mypy/semanal_newtype.py

```diff
@@ -114,11 +114,11 @@ def check_newtype_args(self, name: str, call: CallExpr, context: Context) -> Opt
             self.fail(msg, context)
             return None
 
-        old_type = self.api.anal_type(unanalyzed_type)
+        old_type = self.api.anal_type(unanalyzed_type, report_invalid_types=False)
```

Comment:
This looks a bit ad hoc TBH. I however don't have a better proposal. Could you please add a comment why we pass `False` in this particular case?